corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -58,6 +58,15 @@
             }
         };
 
+        var paginationNotice = function (data) {
+            if (data.aaData.length < data.iTotalRecords) {
+                $('#info-message').html("Showing the current page of data. Switch pages to see more data.");
```

Comment:
should this be translated? I'd maybe also change `the current page` to read `a single page`. 
(Aside: this does seem like a bit of a roundabout way of solving the problem that people aren't seeing the pagination controls (if that is the issue), might be worth putting a pager at the top as well?)